### PR TITLE
[ISV-4625] Run operator-release pipeline on community release trigger

### PR DESCRIPTION
**Pull request by jedinym**

### CODE REVIEW

File: ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml [hunk 1]
```diff
@@ -114,6 +123,12 @@
                           resources:
                             requests:
                               storage: 100Mi
-                    - name: registry-credentials
```
Comment (Allda) [inline ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml:117]: This secret is used by the community when submitting IIB build. Is this going to work when switching to a common operator pipeline?

File: ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml [hunk 2]
```diff
@@ -75,9 +84,9 @@
                     git_pull_request_url: $(tt.params.git_pr_url)
                 spec:
                   timeouts:
-                    pipeline: "1h30m0s"
+                    pipeline: "4h15m0s"
```
Comment (mantomas) [inline ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml:87]: Do we need such timeout also for community? There are no additional steps in the ISV when used for community, so the old timing should be OK from my pov.

**Review by mantomas — APPROVED** [review]
One nitpick, other than that LGTM

File: ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml [hunk 3]
```diff
@@ -117,3 +134,12 @@
                     - name: registry-credentials
```
Comment (Allda) [inline ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml:134]: The workspace that you left here is not present in the isv release pipeline. Please update the isv release pipeline in the way it supports this workpace or remove it if it is not needed and can be replaced by a different workspace.

**Review by Allda — APPROVED** [review]
I'll include the necessary changes to a followup PR.

**Pull request merged**